omnigibson/utils/physx_utils.py

```diff
@@ -242,11 +239,12 @@ def create_physx_particleset_pointinstancer(
     instancer_prim.GetAttribute("physxParticle:particleEnabled").Set(enabled)
 
     if is_isosurface:
+        og.log.warning("Creating an instancer that uses isosurface. The rendering of these particles will have a delay of one timestep.")
```

Comment:
can we add the prim path to this instancer to this warning?

Reply:
done